### ISSUE TITLE
Document how to capture the status of partitioning

### ISSUE THREAD

**Issue by awoods187** [cockroachdb/docs#4978]
As discussed in https://github.com/cockroachdb/docs/issues/2957, we want to document internal tables for partitioning use cases. 

It is not documented but users can run this query for tables: 
```SQL
Select * from crdb_internal.partitions 
join crdb_internal.tables on partitions.table_id=tables.table_id 
where tables.name='mytable'
```
It is also not documented but users can run this query for databases: 
```SQL
Select partitions.name 
from crdb_internal.partitions 
join crdb_internal.tables on partitions.table_id=tables.table_id 
where database_name='mydb
```
We should add this to our various docs including https://www.cockroachlabs.com/docs/v19.1/demo-geo-partitioning.html#step-3-provision-vms

**Comment by jseldess** [comment]
@awoods187, can you add some context about why and when you would use this?

**Comment by awoods187** [comment]
Developers often set up scripts. We know of several customers who wanted to know how to capture the status of partitioning as a part of scripting their environment. Instead of using the new show statements, as they are porcelain, we will recommend they gain information on partitions from this source.

**Comment by ericharmeling** [comment]
This was fixed by #4978.
